README.md

```diff
@@ -84,22 +84,6 @@ alias Noether.List
 
 Easier to read, less verbose, and it encapsulates the handling of `{:ok, _}` tuples. You can focus on writing actual logic instead of repeating the same pattern matches every time.
 
-After looking at `Maybe` and `Either`, let's take a look at `List`. Suppose you have two lists of numbers you want to sum in order, just like this:
```

Comment:
No more mentions of the List module in the readme?

Reply:
I share your concern, however we show `List.sequence()` a few lines above.